tldw_Server_API/app/services/file_artifacts_export_gc_service.py

```
@@ -0,0 +1,123 @@
+                export_format=row.get("export_format"),
+                export_storage_path=None,
+                export_bytes=row.get("export_bytes"),
+                export_content_type=row.get("export_content_type"),
+                export_job_id=row.get("export_job_id"),
+                export_expires_at=None,
+                export_consumed_at=None,
+            )
+            cleared += 1
+        except KeyError:
+            logger.debug(f"files_export_gc: file artifact not found for {file_id}")
+        except Exception as exc:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)